managers/templates/managers/partials/display_project_expected_hours.html

```diff
@@ -0,0 +1,31 @@
+{% load data_display_filters %}
```

Comment:
Name of this template is not appropriate to what it consists. Template is not a function, it does not `display` or `create` anything. Name of template should consists description what it's have - `list`, `form` or anything else what would be in template.

Reply:
Created new folder `templatetags` in managers app. Necessary tags were added to created  `data_display.py` file in  `templatetags`  directory.

Reply:
Please see https://github.com/Code-Poets/sheetstorm/pull/612#discussion_r659376872. I was talking here about `display_project_expected_hours.html` not about this `data_display_filters` :)

Reply:
merged this file with project_expected_hours.html